.github/workflows/r.yml

```diff
@@ -42,6 +43,8 @@ jobs:
         if: always()
         shell: Rscript {0}
         run: |
+          remotes::install_cran("devtools")
```

Comment:
A comment: We could extract this to its own R script, but I guess that it's not a priority right now

Reply:
Thanks. Yes, good idea! No time right now but I will add it for the next version backlog.